packages/drivers/odsp-driver/src/graph.ts

```diff
@@ -97,7 +103,7 @@ export async function graphFetch(
             SharingLinkScopeFor.nonFileDefaultUrl,
             siteUrl,
         );
-    const url = graphUrl.startsWith("http") ? graphUrl : `https://graph.microsoft.com/v1.0/${graphUrl}`;
+    const url = graphUrl.startsWith("http") ? graphUrl : `${msGraphOrigin}/v1.0/${graphUrl}`;
```

Comment:
Method already supports passing full url. Can it be used to accomplish the same result?